.circleci/config.yml

```diff
@@ -1417,6 +1417,14 @@ jobs:
 
             chmod a+x .jenkins/pytorch/macos-test.sh
             unbuffer .jenkins/pytorch/macos-test.sh 2>&1 | ts
+      - run:
+          name: Report results
+          no_output_timeout: "5m"
+          command: |
+            set -ex
+            pip install typing_extensions boto3
```

Comment:
probably package it in a shell script and at the beginning run `macos-common.sh` ?

Reply:
I'll try that next, right now I wonder if having just the source conda activate command would work